core/src/test/java/io/questdb/test/cutlass/http/ExpParquetExportTest.java

```diff
@@ -465,6 +465,23 @@ public void testExportDropConnection() throws Exception {
                 }));
     }
 
+    @Test
+    public void testExportWithAddColumn() throws Exception {
+        getExportTester()
```

Comment:
getExportTester() is misplaced, it should be in this class, not the base class - nothing uses it outside of this class

Reply:
fixed